common/middleware/set-period.js

```diff
@@ -0,0 +1,6 @@
+function setPeriod(req, res, next, period) {
```

Comment:
I wonder if this should be renamed to `setDatePeriod` now its in a more common place. It feels like "period" on its own isn't as clear now its not contained within the moves app.